docs/general/builtins/scalar-functions.rst

```diff
@@ -2446,6 +2446,46 @@ null_string
 If the ``null_string`` argument is omitted or NULL, none of the substrings of
 the input will be replaced by NULL.
 
+.. _scalar-array-min:
+
+``array_min(anyarray)``
+---------------------------------------------------------
```

Comment:
```suggestion
``array_min(array)``
--------------------
```

switched the argument to ``array`` for improved clarity and to match the subsequent text